Duke.java

```diff
@@ -0,0 +1,216 @@
+import javax.management.InstanceNotFoundException;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Duke {
+    private String name = "Lakinta";
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<String> tasks = new ArrayList<>();
+    private ArrayList<Task> taskArrayList = new ArrayList<>();
+    private int count = 1;
+    private final String filePath = "./data/duke.txt";
+
+    public Duke() {
+        // Load tasks from the file when the chatbot starts up
+        loadTasksFromFile();
+    }
+
+
+    public void greeting() {
```

Comment:
I think as for the naming convention, methods' names should be a verb so maybe you can consider greet() instead? (this is quite nitpicking though sorry haha)